src/App.tsx

```diff
@@ -1,156 +1,207 @@
+/* eslint-disable no-console */
+/* eslint-disable quote-props */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import React, { useEffect, useState } from 'react';
+import { Authorization } from './components/authorization';
+import { Etodos, LogingSteps, ResponseError } from './types/enum';
+import { UserData } from './types/userData';
+import { Todo } from './types/Todo';
+import {
+  deleteTodoOnServer,
+  getTodosFromServer,
+  setTodoCompleteStatus,
+} from './api';
+import { TodoItem } from './components/Todo';
+import { TodoHeader } from './components/TodoHeader';
+import { Footer } from './components/Footer';
+import { Notification } from './components/Notification';
+
+const defaultUser = {
+  createdAt: '2023-07-10T13:09:53.578Z',
+  email: 'gookidoo@gmail.com',
+  id: 11038,
+  name: 'Віктор Булденко',
+  phone: null,
+  updatedAt: '2023-07-10T13:09:53.578Z',
+  username: null,
+  website: null,
+};
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
+  const [step, setStep] = useState(LogingSteps.EMAIL);
+  const [user, setUser] = useState<UserData>(defaultUser);
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [uncomplete, setUncomplete] = useState<number>(0);
+  const [showFooter, setShowFooter] = useState<boolean>(false);
```

Comment:
Fix boolean variable naming according [tips on naming boolean variables](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig). Fix it everywhere
```suggestion
  const [isShowFooter, setIsShowFooter] = useState<boolean>(false);
```